File: lecture1/mario/mario.c
#include<stdio.h>
#include<cs50.h>




int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    // make all characters spaces

    char row_status[height + 2 + height];
    for (int i = 0; i < height + 4; i++)
    {
        row_status[i] = ' ';
    }
    for (int row = 1; row <= height; row++)
    {
        // add hashes and end the string
        row_status[height - row] = '#';
        row_status[height + 1 + row] = '#';
        row_status[height + 2 + row] = 0;

        // print out characters
        for (int i = 0; i < height + 2 + row; i++)
        {
            printf("%c", row_status[i]);
        }
        printf("\n");
    }


}

File: lecture6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
height = get_int("Height: ")
# make sure height is between 1 and 8 inclusive
while height > 8 or height < 1:
    height = get_int("Height: ")

for i in range(1, height + 1):
    # print spaces at the beginning
    for j in range(height - i):
        print(" ", end="")
    # print hashes at the beginning
    for k in range(i):
        print("#", end="")
    # print space in middle
    print("  ", end="")
    # print final hashes
    for x in range(i):
        print("#", end="")
    # print ending whitespace and new line
    print()